src/toolbars/toolbar-component.js

```diff
@@ -52,6 +52,7 @@ angular.module('patternfly.toolbars').component('pfToolbar', {
 
     function viewSelected (viewId) {
       ctrl.config.viewsConfig.currentView = viewId;
+      ctrl.config.isTableView = viewId === 'tableView';
```

Comment:
It's either we use the existing viewsConfig, or introduce a new config property like 'isTableView: true'